pruner/find.go

```diff
@@ -18,7 +18,14 @@ func (s *Service) findPruneableHeaders(
 	ctx context.Context,
 	lastPruned *header.ExtendedHeader,
 ) ([]*header.ExtendedHeader, error) {
-	pruneCutoff := time.Now().UTC().Add(-s.window)
```

Comment:
The goal here is to align the logic with the header pruning one as described in the respective CIP.